whole-line-or-region.el

```diff
@@ -81,6 +81,12 @@
   :group 'editing-basics
   :group 'convenience)
 
+(defcustom whole-line-or-region-visual-p nil
+  "Whether or not visual lines are targeted."
+  :group 'editing-basics
+  :group 'convenience
```

Comment:
```suggestion
  :group 'editing-basics
  :group 'convenience
```
You can remove these, then this custom var will naturally get put in the group above, which is what we'd really want.